### PR TITLE
IMM Service addition

### PR DESCRIPTION
**Pull request by twurman**

### CODE REVIEW

File: imm/common/detect.cpp [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+			(double)runtimesearch / totaltime);
+	}
+	if(analyze){
+		printf("%.2f,%.2f,%.2f\n", (double)totalfeat / 1000, (double)totaldesc / 1000, (double)runtimesearch / 1000);
+	}
+	return trainImgs->at(bestIdx);
+}
+
+/*
+po::variables_map parse_opts(int ac, char **av) {
```
Comment (jhauswald) [inline imm/common/detect.cpp]: probably best to remove any code thats commented out

Comment (twurman) [inline imm/common/detect.cpp]: done

**Comment by twurman** [comment]
Don't approve this until I finish changing the thrift interface to a standard name

**Comment by jhauswald** [comment]
@twurman you might as well make a new pull request from a clean branch then, close it if you're doing that

**Pull request closed without merging**